docs/dev/reports/README.md

```diff
@@ -0,0 +1,469 @@
+    timestamp:
+      nanos: 0
+      seconds: 1681659534
+  - category: Pod Security Standards (Baseline)
+    message: validation rule 'app-armor' passed.
+    policy: restrict-apparmor-profiles
+    result: pass
+    rule: app-armor
+    scored: true
+    severity: medium
+    source: kyverno
+    timestamp:
+      nanos: 0
+      seconds: 1681659473
+  - category: Pod Security Standards (Baseline)
+    message: validation rule 'check-seccomp' passed.
+    policy: restrict-seccomp
+    result: pass
+    rule: check-seccomp
+    scored: true
+    severity: medium
+    source: kyverno
+    timestamp:
+      nanos: 0
+      seconds: 1681659473
+  - category: Pod Security Standards (Restricted)
+    message: validation rule 'check-seccomp-strict' anyPattern[1] passed.
+    policy: restrict-seccomp-strict
+    result: pass
+    rule: check-seccomp-strict
+    scored: true
+    severity: medium
+    source: kyverno
+    timestamp:
+      nanos: 0
+      seconds: 1681659534
+  - category: Pod Security Standards (Baseline)
+    message: validation rule 'check-sysctls' passed.
+    policy: restrict-sysctls
+    result: pass
+    rule: check-sysctls
+    scored: true
+    severity: medium
+    source: kyverno
+    timestamp:
+      nanos: 0
+      seconds: 1681659473
+  - category: Pod Security Standards (Restricted)
+    message: validation rule 'restricted-volumes' passed.
+    policy: restrict-volume-types
+    result: pass
+    rule: restricted-volumes
+    scored: true
+    severity: medium
+    source: kyverno
+    timestamp:
+      nanos: 0
+      seconds: 1681659534
+  summary:
+    error: 0
+    fail: 0
+    pass: 19
+    skip: 0
+    warn: 0
+```
+
+That's 6.28KB of json data per report.
+If you have 1000 pods in a cluster, it gives:
+- 1000 admission reports (aggregated) - 6.28KB * 1000 ~ 6.28 MB
+- 1000 background scan reports - 6.28KB * 1000 ~ 6.28 MB
+
+With autogen you have to account for `Deployment`, `StatefulSet`, `Job`, `CronJob`, `ReplicaSet` too (reports for them will be approximately the same size).
+
+And of course ephemeral admission reports consume space too, even if they should be cleaned up quickly.
+
+Finally you need to take policy reports into account too.
+Basically they should consume approximately the same disk space as background scan reports (approximately because if some policies don't run in the background, results will come from aggregated admission reports).
+
+The number of policy reports is usually inferior to the number of other reports but they are bigger in size.
+
+In the end, to use PSS policies (with `restricted` profile) on a 1000 pods cluster, you should budget around 25MB of etcd storage, spread across ~2000 reports.
+
+In comparison, an average `Pod` is around 5KB (at least for a pod with a single container). The storage for 1000 pods is around 5MB. We can see that reports has a significant stoage impact. Fortunately ETCD stores data on disk and associated cost should be neglictible.
```

Comment:
The concerns are:
1. The 8GB etcd limit and the ramifications of exceeding that (see https://aws.amazon.com/blogs/containers/managing-etcd-database-size-on-amazon-eks-clusters/)
2. The load and performance impacts on etcd due to reporting